extensions/VisualEditor/wikia/modules/ve/init/ve.init.wikia.ViewPageTarget.init.js

```diff
@@ -229,7 +229,7 @@
 
 			if ( history.pushState && uri.query.veaction !== 'edit' ) {
 				if (!pageExists) {
```

Comment:
I would leave it as is - we didn't want to change much of VE code in the first place.